.dependabot/config.yml

```diff
@@ -0,0 +1,9 @@
+version: 1
+update_configs:
+  - package_manager: "go:modules"
+    directory: "/"
+    update_schedule: "daily"
+    ignored_updates:
```

Comment:
Sorry, I think it's better to have `allowed_updates` with `github.com/giantswarm/*`. but we can change after this.

Reply:
IMO external updates are as important if not more important than internal ones in general. Of course you can tune this, but external components introduce vulnerabilities and these should be monitored

Reply:
Yes happy to try it with just `k8s.io` ignored. We can iterate if we need to add more later.